### PR TITLE
5.1.0 release

### PR DESCRIPTION
**Pull request by ehoner**
Brooklin release. (New tag and release will be created following merge.)

* Version 5.2.0-SNAPSHOT by @ehoner in https://github.com/linkedin/brooklin/pull/923
* Add support for post datastream create,update,delete and state change action by @hshukla in https://github.com/linkedin/brooklin/pull/915
* Version 4.2.0-SNAPSHOT by @ryannedolan in https://github.com/linkedin/brooklin/pull/914
* Update zookeeper dependency to import from LinkedIn published version by @surajkn in https://github.com/linkedin/brooklin/pull/917
* Set datastream status to deleting for delete call by @hshukla in https://github.com/linkedin/brooklin/pull/918
* Bump version after release by @hshukla in https://github.com/linkedin/brooklin/pull/920
* BMM Restart Improvements Part 1. Leader Coordinator Issuing Assignment Tokens by @jzakaryan in https://github.com/linkedin/brooklin/pull/919

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
 # Change Log
-All notable changes to this project will be documented in this file.  
+All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 5.1.0 - 2023-01-19
```
Comment (hshukla) [inline CHANGELOG.md:5]: FYI: there were three rereleases in between, which we did not document well. I don't think that would really matter much for description as you have already covered it here.

**Review by hshukla — APPROVED** [review]

**Review by ryannedolan — APPROVED** [review]
We seem to be bumping major versions frequently. It may be useful to call out why this is a major bump, cuz I cannot really tell from the changelog.

File: CHANGELOG.md (hunk 1 above)
Comment (ehoner) [reply CHANGELOG.md:5]: Thanks @hshukla, I will mention in the release notes that the prior releases were not listed.

**Comment by ehoner** [comment]
> We seem to be bumping major versions frequently. It may be useful to call out why this is a major bump, cuz I cannot really tell from the changelog.

It was changed with the zookeeper dependency change https://github.com/linkedin/brooklin/pull/917. I will add that explanation too.

**Pull request merged**